README.md

```diff
@@ -229,29 +75,29 @@ Contributing to Amber can be a rewarding way to learn, teach, and build experien
 
 Amber is a community effort and we want You to be part of it. [Join Amber Community!](https://github.com/amberframework/amber/blob/master/.github/CONTRIBUTING.md)
 
-1. Fork it (https://github.com/amberframework/amber/fork)
-2. Create your feature branch (git checkout -b my-new-feature)
-3. Write and execute specs (crystal spec)
-4. Commit your changes (git commit -am 'Add some feature')
-5. Push to the branch (git push origin my-new-feature)
+1. Fork it https://github.com/amberframework/amber/fork
+2. Create your feature branch `git checkout -b my-new-feature`
+3. Write and execute specs `crystal spec`
+4. Commit your changes `git commit -am 'Add some feature'`
+5. Push to the branch `git push origin my-new-feature`
 6. Create a new Pull Request
 
-## Amber Core Team
+## Contributors
 
-- Dru Jensen [drujensen](https://github.com/drujensen) 
-- Elias Perez [eliasjpr](https://github.com/eliasjpr) 
-- Isaac Sloan [elorest](https://github.com/elorest) 
-- Faustino Aguilar [faustinoaq](https://github.com/faustinoaq) 
-- Nick Franken [fridgerator](https://github.com/fridgerator)
-- Mark Siemers [marksiemers](https://github.com/marksiemers)
-- Robert Carpenter [robacarp](https://github.com/robacarp) 
+- [Dru Jensen](https://github.com/drujensen "drujensen")
+- [Elias Perez](https://github.com/eliasjpr "eliasjpr")
+- [Isaac Sloan](https://github.com/elorest "elorest")
+- [Faustino Aguilar](https://github.com/faustinoaq "faustinoaq")
+- [Nick Franken](https://github.com/fridgerator "fridgerator")
+- [Mark Siemers](https://github.com/marksiemers "marksiemers")
+- [Robert Carpenter](https://github.com/robacarp "robacarp")
 
-See more [Amber contributors](https://github.com/amberframework/amber/graphs/contributors)
+[See more Amber contributors](https://github.com/amberframework/amber/graphs/contributors)
 
 ## License
 
 This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details
 
 ## Acknowledgments
 
-* Inspired by Kemal, Rails, Phoenix, Hanami
+* Inspired by [Kemal](https://kemalcr.com/), [Rails](https://rubyonrails.org/) [Phoenix](https://phoenixframework.org/) and [Hanami](https://hanamirb.org/)
```

Comment:
Keep all the commas:
* Inspired by [Kemal](https://kemalcr.com/), [Rails](https://rubyonrails.org/), [Phoenix](https://phoenixframework.org/), and [Hanami](https://hanamirb.org/)